solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1319,7 +1321,125 @@ describe("RandomBeacon - Group Creation", () => {
           await assertDkgResultCleanData(randomBeacon)
         })
       })
+
+      context("when the approver is different than the submitter", async () => {
```

Comment:
I'd consider moving these tests under `with off-chain dkg time passed -> with dkg result submitted -> wth challenge period passed` context. Note that we already have there `called by a third party` block of unit tests. We really need to have there two: `when called by the DKG result submitter` and `when called by a third party`.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2709/commits/ca694fea2c0632d082454df62c51df640d3bb23d.